GETTING_STARTED.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started
+
+## Table of Contents
+
+- [Parrot Security](#parrot)
+
+## Parrot
+
+Parrot Security is a Debian-based Linux distribution designed for cybersecurity. It is lighter than other security distros and comes with many pre-installed penetration testing and privacy tools. Follow this guide to setup [Parrot Security](https://www.parrotsec.org/) with Docker on Oracle VirtualBox. Similar steps can be taken to setup [Kali Linux](https://www.kali.org/) instead.
+
+### Downloads
+
+- Download [Parrot Security OVA](https://www.parrotsec.org/download/) virtual machine image and load it on VirtualBox
+
+- Download and install [Oracle VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+
+### Virtual Machine Setup
+
+- Launch VirtualBox and make a **New** virtual machine. Choose a name and select Linux in *Type* and Debian (64-bit) in *Version*.
+
+- Assign resources according with [Parrot Security system requirements](https://docs.parrotlinux.org/info/system-requirements/) (e.g at least 2 GB of RAM and 40 GB hard drive)
+
+- Create a Dynamically allocated Virtual Hard Disk and select VHD.
+
+- Once created, use the Settings -> System tab to make sure *Optical* is first in Boot Order
+
+- Also in Settings, but in the Storage tab, select *Add Optical Drive* (the disk  :cd: icon next to *Controller: IDE*) and select the Parrot Security image you downloaded previously.
+
+### Parrot Security Installation
+- Launch your virtual machine and follow the installer's instructions. The *Install with GTK GUI* option is recommended for ease of use. You should be ready to boot Parrot Security OS afterwards.
+
+### Docker Installation
+
+- Inside your Parrot OS, [Install Docker Engine on Debian
+](#docker-debian) with **one important change**. Add the repository to sources list by:
+
+    `$ sudo nano /etc/apt/sources.list`
+
+    Appending:
+
+    `deb [arch=amd64] https://download.docker.com/linux/debian buster stable`
+
+    Note: You may get prompts about overwriting parameters on dependencies. **Ignore** all changes.
+
+### References
+- ###### [docker-debian](https://docs.docker.com/engine/install/debian/)
```

Comment:
These links don't appear in compiled markdown, so you don't need the references section.
Nevermind though: as there are a few other links in this repo, it would be better to revert back to how you had it - sorry

Reply:
no worries, done